hr_timesheet_sheet_attendance/tests/test_hr_attendance.py

```diff
@@ -0,0 +1,62 @@
+from odoo.addons.hr_timesheet_sheet_attendance.tests.\
+    hr_timesheet_sheet_test_cases import HrTimesheetTestCases
+from odoo.exceptions import ValidationError, UserError
+
+
+class TestHrAttendance(HrTimesheetTestCases):
+
+    def setUp(self):
+        super(TestHrAttendance, self).setUp()
+        self.attendance_1 = self._create_attendance(
+            employee=self.employee,
+            checkIn='2018-12-12 10:00:00'
+        )
+
+    def test_00_compute_sheet_id(self):
+        # check sheet_id in attendances
+        self.attendance_1._compute_sheet_id()
+        self.assertEqual(
+            self.timesheet,
+            self.attendance_1.sheet_id,
+            "Error while computing sheet_id on attendance.\
+            \nMethod: _compute_sheet_id"
+        )
+
+    def test_01_test_timezone_conversion(self):
+        # check for _get_attendance_employee_tz
+        self.user_id.tz = 'Etc/GMT+12'
+        date = '2018-12-12 10:00:00'
```

Comment:
same here

Reply:
yes. but we this method is doing same as it convert to datetime.object > change timezone > return string in odoo formate, that's why there is no need to convert it to DEFAULT_SERVER_DATETIME_FORMAT format